### ISSUE TITLE
[QUESTION]: Can we have different weights for each collection while doing a similarity search?

### ISSUE THREAD

**Issue by ricksaha2000** [milvus-io/pymilvus#2189]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What is your question?

I had one other use case that I am facing challenges on, Suppose we have "n" collections, I want to do a single similarity search on these two collections on a common vector field. Now I want to give weights to each of these collections which will bring affect to the search results. Is there anything for this type of scenarios?

**Comment by XuanYang-cn** [comment]
@ricksaha2000 Yes, hybrid_search is what you're looking for. You could add multiple vector fields in one collection, and search them back based on custom weigths.

Ref: https://milvus.io/docs/multi-vector-search.md#Step-2-Configure-a-Reranking-Strategy

Weighted hybrid search is only available inside one collection, search between collections are not supported yet.
